Traceback (most recent call last):
  File "/faststorage/project/CancerEvolution/Projects/janne_cns/cns/scripts/gensimHDP.py", line 12, in <module>
    file = pd.read_table(args.featurefile, dtype=str)
  File "/home/janneae/miniconda3/envs/cns/lib/python3.10/site-packages/pandas/util/_decorators.py", line 311, in wrapper
    return func(*args, **kwargs)
  File "/home/janneae/miniconda3/envs/cns/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 779, in read_table
    return _read(filepath_or_buffer, kwds)
  File "/home/janneae/miniconda3/envs/cns/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 575, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/janneae/miniconda3/envs/cns/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 933, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/home/janneae/miniconda3/envs/cns/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1217, in _make_engine
    self.handles = get_handle(  # type: ignore[call-overload]
  File "/home/janneae/miniconda3/envs/cns/lib/python3.10/site-packages/pandas/io/common.py", line 789, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: '../steps/discretized_10.features'
